drivers/input/Kconfig

```diff
@@ -622,6 +622,25 @@ config INPUT_BUTTONS_NPOLLWAITERS
 
 endif # INPUT_BUTTONS
 
+config INPUT_SBUTTON
```

Comment:
Maybe add `KEY_THRESH_MS` as configurable value?

Reply:
done (I'm updating locally and marking as Resolved to simplify my review)